pallets/manta-pay/src/lib.rs

```diff
@@ -474,23 +488,35 @@ pub mod pallet {
         }
     }
 
-    impl<T> From<SenderPostError<SenderLedgerError>> for Error<T> {
+    impl<T> From<SenderPostError<SenderLedgerError>> for Error<T>
+    where
+        T: Config,
+    {
         #[inline]
         fn from(err: SenderPostError<SenderLedgerError>) -> Self {
             match err {
                 SenderPostError::AssetSpent => Self::AssetSpent,
                 SenderPostError::InvalidUtxoAccumulatorOutput => Self::InvalidUtxoAccumulatorOutput,
-                SenderPostError::UnexpectedError(_) => Self::InternalLedgerError,
+                SenderPostError::UnexpectedError(_) => {
+                    let _ = T::Suspender::suspend_manta_pay_execution();
```

Comment:
todo: should we handle this result ?

Reply:
Maybe dropping a `log::error`  would be the way to go. 

Not much we can do if it fails